org.cockpit_project.CockpitClient.metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is the one from upstream, automatically generated by a script which lives in the git repository.  It doesn't get included as part of the tarball, however: the reason for that is the version history.  It's easier to keep track of that here. when we push the new release, instead of having to explicitly write a commit for each version upstream.

Reply:
it should be part of the tarball. and you should use the one shipping when that happen.

Reply:
Yeah, this guideline is intended to make sure that the whole Linux ecosystem gets better since we absolutely do not live in an 'ideal' world where flatpak is the only way of getting apps! I do understand some of the potentially annoying workflow issues this can put into 'doing a release' though.